src/app/Sankalan/Quiz/problems.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use \DUCS\Sankalan\Database;
+
+/**
+ * returns problems json based on event
+ * @param  string $event event id
+ * @return array        list of problems
+ */
+function getProblems($event_id)
+{
+    $db = new Database();
+    $event = $db->query('SELECT name FROM events WHERE event_id = :event_id', ['event_id' => $event_id], true);
+
+    if (!$event) die("Invalid ID");
```

Comment:
`die` is not a good solution here. what we can do is to return an error code as response